### ISSUE TITLE
Investigate CircleCI parallelization of Unit Test

### ISSUE THREAD

**Issue by hinzed1127** [hinzed1127/pivotal-to-gh-importer#421]
This issue was imported from Pivotal Tracker.

  | Field | Value |
  | --- | --- |
  | Story ID | [#183393750](https://www.pivotaltracker.com/story/show/183393750) |
  | State | accepted |
  | Requested By | Michael Deeb |
  | Owned By | JP Hechter |
  | Estimate |  |
  | Priority | p3 - Low |
  | Created At | Sep 27, 2022 |
  | Accepted At | Mar 15, 2023 |
      

## Comments
Circle CI provides a param `parallelism` that will create identical duplicates of a given environment, and "test splitting" which will attempt to intelligently divide tests amongst those environments to be executed in parallel. I'm currently trying to use these to run our unit tests more efficiently.

I was running into a couple stumbling blocks with the jest test runner recognizing each of our projects, and currently the workflow is only running about 440 tests. While it would be more scalable to use the features provided by Circle CI to run our tests, there might be more immediate payoff by manually dividing them. (JP Hechter - Mar 10, 2023)

I changed the match string and I'm up to 1752 tests. This is closer to the true number, but it's not entirely right. Need to fool around with this more.

Additionally, all of the runners are reporting errors in Circle CI, but are showing 0 failed tests. need to figure out what that is. (JP Hechter - Mar 10, 2023)
